tinkergraph-gremlin/src/test/java/org/apache/tinkerpop/gremlin/tinkergraph/structure/io/graphson/AbstractTinkerGraphGraphSONTranslatorProvider.java

```diff
@@ -242,12 +242,24 @@ public GraphTraversalSource traversal(final Graph graph) {
         return g.withStrategies(new TranslationStrategy(g, new GraphSONTranslator<>(JavaTranslator.of(g), version), true));
     }
 
+    @Graph.OptOut(
+            test = "org.apache.tinkerpop.gremlin.process.traversal.step.OrderabilityTest",
+            method = "g_inject_order",
+            reason = "GraphSONv2 does not properly round trip Maps and Sets")
```

Comment:
This might not be important for 3.6.x since GraphSonV3 is supposed to be used here but we still haven't deprecated graphsonv2 hence, it requires some attention. Please create a JIRA for this. We will have to investigate what's wrong with graphsonv2 and document the shortcoming.